src/components/mortgage-investment-compare/input-card.js

```diff
@@ -16,18 +17,57 @@ import {
 import { toUSD } from '../../utils/numberFormat'
 
 /**
+ *
  * @param {*} props
- *   @property {number} loanAmt
- *   @property {number} investmentRate
- *   @property {number} inflation
- *   @property {function} onInputChange
- *   @property {Object} option1
- *   @property {Object} option2
- *   @property {Object} shorterOption
- *   @property {Object} longerOption
+ *   @property {number} mustBeGreaterThan - If present, number must be greater
  */
+function ValidatedNumberInput (props) {
+  const [value, setValue] = useState(props.value)
+  const [isValid, setIsValid] = useState(true)
+
+  function onInputChange (event) {
+    const value = event.target.value
+    setValue(value)
+    const parsed = parseFloat(value)
+    if (!value || !_.isNumber(parsed)) {
+      setIsValid(false)
+      return
+    }
+    if (
+      _.isNumber(props.mustBeGreaterThan) &&
+      parsed <= props.mustBeGreaterThan
```

Comment:
having a `props.mustBeGreaterThan` is great but for the most part, we can pass the `min` attribute to `<Input>` below and prevent the user from going below 0 altogether.  In this case you'd pass `min={0}` when applicable. 

https://www.w3schools.com/tags/att_input_min.asp